fsspec_reference_maker/grib2.py

```diff
@@ -187,16 +188,17 @@ def example_multi(filter={'typeOfLevel': 'heightAboveGround', 'level': 2}):
         with open(os.path.basename(url).replace("grib2", "json"), "w") as f:
             json.dump(out, f)
 
-    # stitch with
-    # files = ['hrrr.t22z.wrfsfcf01.json',
-    #  'hrrr.t23z.wrfsfcf01.json',
-    #  'hrrr.t00z.wrfsfcf01.json',
-    #  'hrrr.t01z.wrfsfcf01.json',
-    #  'hrrr.t02z.wrfsfcf01.json',
-    #  'hrrr.t03z.wrfsfcf01.json',
-    #  'hrrr.t04z.wrfsfcf01.json',
-    #  'hrrr.t05z.wrfsfcf01.json',
-    #  'hrrr.t06z.wrfsfcf01.json']
-    # mzz = MultiZarrToZarr(files, remote_protocol="s3", remote_options={"anon": True}
-    #                       concat_kwargs={"dim": 'time'})
-    # mzz.translate("hrrr.total.json")
+
+def example_combine():
+    files = ['hrrr.t22z.wrfsfcf01.json',
```

Comment:
```suggestion
    from fsspec_reference_maker.combine import MultiZarrToZarr
    files = ['hrrr.t22z.wrfsfcf01.json',
```

Reply:
Yep, this was only run in an interactive session, as you probably guessed.